integration/log_integration_test.sh

```diff
@@ -19,7 +19,7 @@ popd > /dev/null
 
 # Set an exit trap to ensure we kill the RPC server once we're done.
 trap "kill -INT ${RPC_SERVER_PID}" EXIT
-sleep 2
+sleep ${STARTUP_WAIT_SECONDS}
```

Comment:
You could also do:
`wget --retry-connrefused --waitretry=1 -t $(STARTUP_WAIT_SECONDS} localhost:${RPC_PORT}`

So that it continues as soon as the port is opened.